doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -52,6 +52,22 @@ Build system
 
   * Manifest file entry ``mbedtls`` (:makevar:`ZEPHYR_MBEDTLS_MODULE_DIR`) checked out at path :file:`modules/crypto/mbedtls` now points to |NCS|'s fork of MbedTLS instead of Zephyr's fork.
 
+Board support
+-------------
+
+* nRF52 and nRF53 series development kits:
+
+  * Added :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` and :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU_SPEEDUP` to configure FOTA updates.
+
+* Thingy:53:
+
+  * Simplified board configuration:
+
+    * Added :kconfig:option:`CONFIG_BOARD_SERIAL_BACKEND_CDC_ACM` and DTS `cdc_acm_uart` node to configure USB CDC ACM to be used as logger's backend by default.
+    * Provided support for Thingy:53 FOTA updates within the :kconfig:option:`CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU` option.
```

Comment:
If the MCUboot bootloader was enabled in this release, it would be great to mention it as a separate entry. @MarekPieta

Reply:
Judging by the commit date 28.11.2021, MCUboot bootloader was enabled for Thingy:53 already for NCS 2.2, but Marek please confirm just in case.